### PR TITLE
Fix PrepareUIRequest+ new theme locations

### PR DESCRIPTION
**Pull request by gpickin**

### CODE REVIEW

**Comment by gpickin** [comment]
https://ortussolutions.atlassian.net/browse/CONTENTBOX-986

File: modules/contentbox/models/system/CBHelper.cfc [hunk 1]
```diff
@@ -369,7 +369,7 @@ component accessors="true" singleton threadSafe{
 
 		// Do we set a layout for them already?
 		if( structKeyExists( arguments, "layout" ) ){
-			event.setLayout( name="#prc.cbTheme#/layouts/" & arguments.layout, module="contentbox" );
+			event.setLayout( name="#prc.cbThemeRecord.includePath#/layouts/" & arguments.layout );
```
Comment (lmajano) [inline modules/contentbox/models/system/CBHelper.cfc]: I am not sure this will work.  Just use :`setLayout( name="", module="" )` instead

Comment (gpickin) [reply modules/contentbox/models/system/CBHelper.cfc]: it works fine... i tested it on my lucee.
Should i use module equal nothing?

Comment (lmajano) [reply modules/contentbox/models/system/CBHelper.cfc]: Why would the module be empty?

Comment (gpickin) [reply modules/contentbox/models/system/CBHelper.cfc]: If you set a module, it looks in that module for the layout. 
So leaving it blank, seems to look in the root of the app.

What do you think?

Comment (lmajano) [reply modules/contentbox/models/system/CBHelper.cfc]: Ok so I am confused then. Are you saying you want it to look in the root? Or in a module?

**Pull request merged**